schema/flavor.json

```diff
@@ -6,25 +6,38 @@
         "name": {
             "type": "string"
         },
+        "comment": {
+            "type": "string"
+        },
         "type": {
             "type": "string",
             "enum": [
                 "KEM",
-                "PKE",
```

Comment:
I would've thought `PKE` is the more commonly used one. But maybe not.

Reply:
Certainly. However I opted for KTM because it is more precise (in contrast to KEM), and because PKE is also used in contrast to symmetrical encryption...

Reply:
We could name it `PKE/KTM`

Reply:
Don't like that. We can still add some help text in the UI, but I think we shouldn't make identifiers clumsy by including explanations in them...